build.sbt

```diff
@@ -23,31 +23,46 @@ lazy val scala213Version      = "2.13.7"
 lazy val scala30Version       = "3.1.0"
 lazy val slf4jVersion         = "1.7.32"
 
-// These are releases to ignore during MiMa checks
-lazy val botchedReleases = Set("0.8.0", "0.8.1")
+// Basic versioning and publishing stuff
+ThisBuild / tlBaseVersion := "1.0"
+ThisBuild / scalaVersion := scala213Version
+ThisBuild / crossScalaVersions := Seq(scala212Version, scala213Version, scala30Version)
+ThisBuild / developers += tlGitHubDev("tpolecat", "Rob Norris")
+ThisBuild / tlSonatypeUseLegacyHost := false
+ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("11"))
+ThisBuild / githubWorkflowBuildPreamble ++= Seq(
```

Comment:
FYI there's also `githubWorkflowBuild` (the default `ci` step resides in here) and `githubWorkflowBuildPostamble`.

Reply:
Hm yeah maybe I should do docs last.